### ISSUE TITLE
Relink the New/Reborn Open Shell in the Header of the Readme.md

### ISSUE THREAD

**Issue by blackcrack** [coddec/Classic-Shell#34]
you should add a link and a Description to the 
https://github.com/Open-Shell/Open-Shell-Menu

best regards
Blacky

**Comment by coddec** [comment]
@blackcrack 
Thanks for reminding 👍 
https://github.com/coddec/Classic-Shell/blob/master/README.md

**Comment by blackcrack** [comment]
Hi @coddec ,
yes, because nowhere it is a link to the new name in the public ,
and in the german conversation was the peoples a little confused (like i think around in the world),
just because i am so angry about was to change the name 2ice or more several times..
why i was so angry because this nice and really super program.. .. this Program hast make a Treadmake in the brain of the peoples because it was so very good and for free.. and then comes a programmer with cat ears and meant it mus changing something .. and this confuse many peoples on the earth where be in the net . And that disappointment was it was made me angry about .. i was no more deludes about .. how ever.. I hope the name "open shell" is now firmly anchored in the future .. 
the next could be "Libre Shell" ... what for a bullsh... :\

Name change without request from outside is like something.. 
there is a red line and i want not die .. 
or, there is a white line, i want not die .. blubb(<= i guess, some word is internation like a drop in the water or a rice bag in the hidden island what you can imagine  .. so at last also bullshit ..)
who the hack have insert this bullshit information in the brain ? "the name should be changed" \*to curse properly at this point/Right at this point swearing \*



best regards
Blacky

**Comment by coddec** [comment]
Hi @blackcrack 
Thanks for the feedback :)
We've made name changing decision together, we should not blame anyone, I assume now we all know why the name issue exist at start, and why second change.
It was troublesome for all of us, users and developers, sorry for any inconvenience, but it was due to some reason which we all know I guess.
I believe none of us, users and developers, anyone who involved in the project in anyway want fiddle with name pointlessly.
Anyway, it is settle now, don't worry ;)
Keep calm, spread the word and enjoy :)

Thanks for your feedback again 👍 
Have a good one ;)
